clickhouse/results/c6a.metal.json

```diff
@@ -1,58 +1,58 @@
 {
     "system": "ClickHouse",
-    "date": "2022-10-04",
+    "date": "2023-02-14",
     "machine": "c6a.metal, 500gb gp2",
     "cluster_size": 1,
     "comment": "",
 
     "tags": ["C++", "column-oriented", "ClickHouse derivative"],
 
-    "load_time": 137.610,
-    "data_size": 14571706777,
+    "load_time": 461.326,
```

Comment:
This number is much worse than the previous one.
Let's check if the file was decompressed before the insertion.

Reply:
smth wrong with the ec2 instance. I've rerun the benchmark on another vm